src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeCalculationArgumentCollection.java

```diff
@@ -61,7 +62,7 @@ public GenotypeCalculationArgumentCollection clone() {
     /**
      * As of version 4.1.0.0, this argument is no longer needed because the new qual score is now on by default. See GATK 3.3 release notes for more details.
```

Comment:
Does javadoc on arguments get output in our docs at all?

Reply:
Yes, it gets included in the full text description.